notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    pip install diffusers accelerate transformers safetensors sentencepiece huggingface_hub
</pre><p>do we should specify minimal diffusers version? (as I know your code does not</p><p> </p><p>please install in quet mode for reduce logs size.</p><p>please use "" for statement with version specification (such prompt does not work on windows)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done!</p>